File: examples/functions/python/utc-time/handler.py
import json
from datetime import datetime, timezone


def handler(event):
    ctx = event.get("context") or {}
    trigger = ctx.get("trigger") or {}
    now_utc = datetime.now(timezone.utc).isoformat()
    now_local = datetime.now().astimezone().isoformat()

    return {
        "status": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps(
            {
                "function": "utc-time",
                "now_utc": now_utc,
                "now_local": now_local,
                "trigger": trigger,
            },
            separators=(",", ":"),
        ),
    }

File: examples/functions/node/echo/handler.js
exports.handler = async (event) => {
  const query = event.query || {};
  const context = event.context || {};
  const key = query.key ?? null;
  return {
    status: 200,
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({
      key,
      query,
      context: {
        user: context.user || null,
      },
    }),
  };
};
